.yamato/test_versions.metafile

```diff
@@ -0,0 +1,7 @@
+# List of editor versions for standalone-build-test and its dependencies.
+test_editors:
+  - version: 2018.4
+  - version: 2019.4
+  - version: 2020.1
+# Waiting on a barracuda fix, see https://jira.unity3d.com/browse/MLA-1464
+#  - version: 2020.2
```

Comment:
Inference is broken for the built player on 2020.2. Disabled until fixed in a new barracuda release.